app/models/invitation.rb

```diff
@@ -40,6 +40,10 @@ def group
     end
   end
 
+  def group_id
```

Comment:
Can we move the into the serializer to avoid putting extra stuff on the model?

```
def group_id
  object.inviter_id
end
```

or, probably even better:

```
attribute :invitable_id, key: :group_id
```